nexus/types/src/external_api/params.rs

```diff
@@ -1172,23 +1172,32 @@ pub struct InstanceCreate {
     #[serde(default)]
     pub external_ips: Vec<ExternalIpCreate>,
 
-    /// The disks to be created or attached for this instance.
+    /// A list of disks to be attached to the instance.
+    ///
+    /// Each disk can either be attached if it already exists or created along
+    /// with the instance.
```

Comment:
I think expanding on the first line is a good idea, but we could go more information-dense and concrete here to justify the extra words.

```rs
/// A list of disks to be attached to the instance.

/// Disk attachments of type "create" will be created, while those of type
/// "attach" must already exist.
```

https://github.com/oxidecomputer/omicron/blob/6d306003f79cfe13ef228a32067da86dacd4a3f0/nexus/src/app/sagas/instance_create.rs#L995-L1000

Reply:
Added!